libcontainer/userns/userns_linux.go

```diff
@@ -12,26 +13,42 @@ var (
 )
 
 // runningInUserNS detects whether we are currently running in a user namespace.
-// Originally copied from github.com/lxc/lxd/shared/util.go
+// Originally copied from github.com/lxc/lxd/shared/util.go.
```

Comment:
This link is gone now. Now it's in here: https://github.com/lxc/incus/blob/main/shared/util.go#L678-L700

Reply:
Thx; updated to use a permalink